.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+Please read EasyPost's general [contribution guidelines](https://github.com/EasyPost/.github/blob/main/CONTRIBUTING.md).
+
+## Code of conduct
+
+Please read EasyPost's [code of conduct](https://github.com/EasyPost/.github/blob/main/CODE_OF_CONDUCT.md).
+
+## Component API best practices
+
+If your pull request involves changes to any component APIs, please read over our list of best practices below.
+
+### One-offs
+
+We define a one-off as a deviation from best practices. We understand that one-offs are virtually inevitable, so when we create them, we should have a good reason for doing so as creating one-offs adds additional context for contributors and users as well as introduces the possibility of creating inconsistent UX.
+
+### Intuitive naming
+
+Property names should be descriptive and intuitive of what they do. Intuitive names are preferred over descriptive names. For example, for a button, `urlLinkDestination` is a descriptive name, but `href` is intuitive.
+
+### Consistent naming
+
+Properties that have identical purposes across different components should share the same name. Similarly, properties that have identical purposes should also share the same type. If one component has a boolean `isValid`, other components that intend to control validity should do the same.
+
+### Property dependency
+
+Although unavoidable at times, we should do our best to not create properties that are dependent upon another. This helps avoid configurations that lead to invalid states. If a component has multiple dependent properties, take a closer look at the overall structure of the component.
+
+### Booleans
+
+- Booleans should be prefixed with simple verbs. For a button, the property name `isActive` is preferred to `active`.
+- By default booleans should be false. This makes the notation a bit cleaner for when the boolean is active:
+
+```JSX
+  // avoid
+  <Button isNotDisabled={false} />
+```
+
+```JSX
+// preferred
+<Button isDisabled />
+```
```

Comment:
this is still showing as differently indented, is that intentional?

<img width="436" alt="image" src="https://user-images.githubusercontent.com/752942/217857454-5bcedfe9-1d1e-4bd4-9912-489656a51e45.png">

Reply:
ah no not intentional, will fix!